packages/cashier/src/hooks/__tests__/useWS.test.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { useWS as useWSShared } from '@deriv/shared';
+import useWS from '../useWS';
+import { TSocketCallTypes } from '../../types';
+
+jest.mock('@deriv/shared');
+
+const mockUseWSShared = useWSShared as jest.MockedFunction<typeof useWSShared>;
+
+const UseWSExample = <T extends keyof TSocketCallTypes>({ name, request }: { name: T; request: any }) => {
+    const WS = useWS(name);
+
+    return (
+        <>
```

Comment:
Normally it is recommended to explicitly define `React.Fragment` in Deriv app codebase

Reply:
@matin-binary Sure I will update 🙇🏻